environment_tests/test-exports-cf/package.json

```
@@ -8,7 +8,7 @@
     "@cloudflare/workers-types": "^4.20230321.0"
   },
   "dependencies": {
-    "@langchain/core": ">=0.2.31 <0.4.0",
+    "@langchain/core": ">=0.2.31 <0.4.0 || ^1.0.0",
```

this dep range wont grab alpha

---

Should we just
```suggestion
    "@langchain/core": "1.0.0-alpha.1",
```

for now?